vendor/modules.txt

```diff
@@ -1,5 +1,7 @@
 # github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d
 github.com/cpuguy83/go-md2man/v2/md2man
+# github.com/mssola/capture v0.0.0-20180102083101-24891570e9b2
+github.com/mssola/capture
```

Comment:
I know this wasn't probably generated by you. I wonder why the other repo is left there commented. Have you tried to run a `go mod tidy` to see if the file is cleaned up?

Reply:
That file is generated by `go mod vendor` and I think it's its default behavior to add a comment the with the repo, version, timestamp and commit id of the repo and then the list of go modules from that repository.
Or are you referring to something different?

Reply:
No worries, it that's how go mod vendor works. Fine with me :)